hack/e2e-suite/services.sh

```diff
@@ -22,11 +22,13 @@ set -o pipefail
 
 KUBE_ROOT=$(dirname "${BASH_SOURCE}")/../..
 
+: ${NAMESPACE:="services-${RANDOM}"}
 : ${KUBE_VERSION_ROOT:=${KUBE_ROOT}}
-: ${KUBECTL:="${KUBE_VERSION_ROOT}/cluster/kubectl.sh"}
+: ${RAW_KUBECTL:="${KUBE_VERSION_ROOT}/cluster/kubectl.sh"}
+: ${KUBECTL:="${RAW_KUBECTL} --namespace=${NAMESPACE}"}
```

Comment:
Please incorporate @lavalamp's fix, which didn't add --namespace until after creating the namespace.